### ISSUE TITLE
Add support for flag returns with no team specified

### ISSUE THREAD

**Issue by Nomad7** [bananahampster/hampalyzer#39]
Currently flag returns appear to be tracked based on the team name indicated in the message that's triggered on the return, for example:
```python
                                case "Blue Flag Returned Message":
                                case "Red Flag Returned Message":
                                    eventType = EventType.FlagReturn;
                                    data.team = Event.parseTeam(nonPlayerDataParts[2].split(" ")[0]);
                                    break;
```
However, in some (ADL) maps this information may not be present, for example this is what triggers on Magelli:
```
L 08/10/2022 - 04:47:12: World triggered "Flag has returned Info"
L 08/10/2022 - 04:47:12: World triggered "#dustbowl_flag_returned"
```
Magelli may be an edge case due to the number of helper entities used in the map, most of which have custom names, but I believe a similar issue affects other one-flag type maps as well. I am happy to add the various customized versions of these flag return messages as they come up, but currently the attempt to parse the team based on the message content results in an error. For example, using the Magelli trigger above:
```
Failed to parse line number 603: World triggered "Flag has returned Info"

file:///[...]/hampalyzer/dist/parser.js:827
                throw error;
                ^
Waiting for the debugger to disconnect...
unknown team: flag
```

**Comment by Nomad7** [comment]
In case it's informative, here's the entity definition for magelli:
```
{
"origin" "3376 1008 120"
"g_e" "1"
"goal_state" "2"
"speak" "vox/deeoo"
"netname" "Flag has returned Info"
"goal_no" "300"
"angles" "0 0 0"
"broadcast" "#dustbowl_flag_returned"
"team_no" "1"
"owned_by" "1"
"classname" "info_tfgoal"
}
```
A similar entity for osaka:
```
{
"origin" "-3072 -3248 -192"
"goal_state" "2"
"b_b" "The Flag has Returned"
"angles" "0 315 0"
"if_goal_is_inactive" "20"
"owners_team_speak" "!CTF_FLAGBACK"
"owned_by" "1"
"team_no" "1"
"goal_no" "100"
"classname" "info_tfgoal"
}
```
Avanti's return entity definition:
```
{
"origin" "2008 -2008 -302"
"angles" "0 315 0"
"broadcast" "#dustbowl_flag_returned"
"if_goal_is_inactive" "11"
"team_no" "2"
"owned_by" "1"
"owners_team_speak" "!CTF_FLAGBACK"
"non_team_speak" "!CTF_EFLAGBACK"
"classname" "info_tfgoal"
}
```
Dustbowl's return entity definition (although dustbowl also has other entities that mention returning and I'm not sure how they overlap):
```
{
"origin" "-2368 2752 -370"
"goal_state" "2"
"angles" "0 315 0"
"broadcast" "#dustbowl_flag_returned"
"if_goal_is_inactive" "11"
"team_no" "2"
"owned_by" "1"
"owners_team_speak" "!CTF_FLAGBACK"
"non_team_speak" "!CTF_EFLAGBACK"
"goal_no" "300"
"classname" "info_tfgoal"
}
```

**Comment by LightTreason** [comment]
Fixed via 513e05e547eeb9d4fe332159d5794434571b8d47 by separating out these ADL capture types into a separate event so that the CTF flag time calculations aren't attempted on the events.
